src/pages/activities/list/list.page.ts

```diff
@@ -107,19 +124,47 @@ export class ActivitiesListPage implements OnInit {
           },
           err => {
             loadingActivities.dismiss().then(() => {
-              loadingFailed.present();
+              if (ACTIVATE_TOAST) {
+                loadingFailed.present();
+              }
             });
           }
         )
   }
+
+  /**
+   * @TODO: remove this feature after development near complete
+   * Prompt user to skip loading to skip forced long wait of API
+   * @param {Function} cb callback if user choose to load API call
+   */
+  promptSkipLoading(cb: Function) {
```

Comment:
@PhilippLaufenberg there is a comment `@TODO: remove this feature after development near complete`
This is the code i added to avoid long loading inevitable wait of `3 x achievement` + `get activity` API requests (practera local server), especially when those api request/waits is not related to some of the frontend development on the other stuff like event, checkin and ranking, which they really added development time if we dont skip those by adding extra code like this.